lib/src/components/servo/servo.dart

```diff
@@ -11,12 +15,25 @@ abstract class Servo extends Resource {
 
   Servo(this.name);
 
+  /// Move the [Servo] to the provided angle.
+  ///
+  /// Args:
+  ///     [angle] (int): The desired angle of the servo in degrees.
```

Comment:
Similar to my [comment on motors](https://github.com/viamrobotics/viam-flutter-sdk/pull/10#discussion_r1184352469), `Args` and `Returns` should be removed

Reply:
✅ thanks for calling this out